R/prep_net_zero_commitments.R

```diff
@@ -16,31 +16,43 @@
 prep_net_zero_commitments <- function(total_portfolio,
                                       peer_group = c("pensionfund", "assetmanager", "bank", "insurance", "other"),
                                       net_zero_targets,
```

Comment:
is this parameter the same as 'fin_data_net_zero_targets' dataset? In that case could we rename it so that it is clear what dataset should be input here?